packages/sanity/src/_internal/cli/actions/build/buildAction.ts

```diff
@@ -75,14 +75,24 @@ export default async function buildSanityStudio(
     spin.succeed()
   }
 
+  const basePath =
+    // Allow `sanity deploy` to override base path
+    overrides?.basePath ||
+    // Environment variables
+    process.env.SANITY_STUDIO_BASEPATH ||
+    // `sanity.cli.ts`
+    cliConfig?.project?.basePath ||
+    // Fallback
+    '/'
+
```

Comment:
Wondering if `??` would be a better operator than `||` here.

Let's say a user wants to reset the base path using env variable:
```sh
# This might not give the expected result, since it should be "/"
# Though we allow "" since we sanitize the value to make sure it ends with a slash
SANITY_STUDIO_BASEPATH="" sanity build
```